### ISSUE TITLE
Adding a scnanned product from Pantry Party to Grocy giving error 400

### ISSUE THREAD

**Issue by canedje** [PantryParty/pantry_party#110]
I just installed the Grocy docker under unraid at my NUC (version 3.0.1) and installed Pantry Party to my Android.
I connect Grocy to Pantry Party by IP and API. (Working fine)
Then if I try to scan a product and filling in the values and saving the product to Grocy it is giving an 400 error saying barcode column not existing in Grocy database and not saving the product

![image](https://user-images.githubusercontent.com/23150947/106369090-f1303780-634e-11eb-996e-bffccf9f6f23.png)

What to do?

**Comment by arjes** [comment]
Hey im sorry. Pantry party was not updated fir grocy 3.0 yet

**Comment by canedje** [comment]
Oke thanks. When will that be?

**Comment by arjes** [comment]
I got started on it, but there was a bug in an underling library that
blocked me. I'll be starting up again when life give me time unfortunately
:(

**Comment by canedje** [comment]
Then I hope this will happen soon😎😁

**Comment by dibyadip-das** [comment]
Getting the same error.

**Comment by nmsgeral** [comment]
Hi, I got same error today.
I guess that there are no updates on this issue.
I noticed that barcode exists in grocy V3 because it is possible to add manually directly on grocy.
However I have no idea how to setup different.
I am totally new with this grocy tool (installed yesterday) and I am trying to learn how to use this tools.
Thanks in advance, looking forward to hear if someone has some solution working.
Best regards
